acos_client/v30/axapi_http.py

```diff
@@ -64,6 +64,16 @@ def request(self, method, api_url, params={}, headers=None,
         LOG.debug("axapi_http: %s url = %s", method, api_url)
         LOG.debug("axapi_http: params = %s", json.dumps(params, indent=4))
 
+        # Update params with **kwargs for currently unsupported configuration
+        # of objects
+        formatted_kwargs = dict([(k.replace('_', '-'), v) for k, v in kwargs.iteritems()])
+        param_keys = params.keys()
+        if params != {}:
```

Comment:
The way it works on the 2.1 side is that it requires the outer wrapper.  i.e. kwargs['axapi_args'] must have "{ virtual-port: { more-stuff: foo } }", and then they're just merged (with axapi_args winning.)